services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+
+from db.models import Genre, Movie, Actor
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
```

Comment:
If your argument can be `None` you should show it in type hint. Also, specify inner type of list. Fix everywhere, please
```suggestion
        genres_ids: list[int] | None= None,
        actors_ids: list[int] | None = None
```